docs/api/tracker-stores.rst

```diff
@@ -36,30 +35,56 @@ SQLTrackerStore
 :Description:
     ``SQLTrackerStore`` can be used to store the conversation history in an SQL database.
     Storing your trackers this way allows you to query the event database by sender_id, timestamp, action name,
-    intent name and typename
+    intent name and typename.
 
 :Configuration:
-    To set up Rasa Core with SQL the following steps are required:
+    To set up Rasa Open Source with SQL the following steps are required:
 
-    1. Add required configuration to your ``endpoints.yml``
+    #. Add required configuration to your ``endpoints.yml``:
 
         .. code-block:: yaml
 
             tracker_store:
                 type: SQL
-                dialect: "sqlite"  # the dialect used to interact with the db
+                dialect: "postgresql"  # the dialect used to interact with the db
                 url: ""  # (optional) host of the sql db, e.g. "localhost"
-                db: "rasa.db"  # path to your db
+                db: "rasa"  # path to your db
                 username:  # username used for authentication
                 password:  # password used for authentication
                 query: # optional dictionary to be added as a query string to the connection URL
                   driver: my-driver
 
-    3. To start the Rasa Core server using your SQL backend, add the ``--endpoints`` flag, e.g.:
+    #. To start the Rasa server using your SQL backend,
+       add the ``--endpoints`` flag, e.g.:
 
         .. code-block:: bash
 
             rasa run -m models --endpoints endpoints.yml
+
+    #. If deploying your model in Docker Compose, add the service to your ``docker-compose.yml``:
+
+           .. code-block:: yaml
+
+              postgres:
+                image: postgres:latest
+
+       To route requests to the new service, make sure that the ``url`` in your ``endpoints.yml``
+       references the service name:
+
+           .. code-block:: yaml
+              :emphasize-lines: 4
+
+                tracker_store:
+                    type: SQL
+                    dialect: "postgresql"  # the dialect used to interact with the db
+                    url: "postgres"
+                    db: "rasa"  # path to your db
+                    username:  # username used for authentication
+                    password:  # password used for authentication
+                    query: # optional dictionary to be added as a query string to the connection URL
+                      driver: my-driver
+
```

Comment:
@ricwo do we know which image one would use to run dynamoDB as a separate service? If so I would add a step here about it (this information used to live [here](https://rasa.com/docs/rasa/user-guide/how-to-deploy/#adding-a-custom-tracker-store) if you want to see the old info)

Reply:
the production version of DynamoDB is not dockerized, and is configured within AWS - I probably wouldn't add any instructions for that, wdyt?

Reply:
makes sense to me 👍